mlir/include/mlir/Dialect/MIOpen/Generator/Conv2dGenerator.h

```diff
@@ -58,8 +61,8 @@ class Conv2dGenerator {
 
   const Config &getConfig() const { return config; }
   void setKernelName(std::string newName);
-
-  int getKernelCount() const;
+  // return gemm id
```

Comment:
this comment is not clear. please state the numbers in the vector returned may NOT be consecutive.